client/webserver/locales/zh-cn.go

```diff
@@ -4,263 +4,679 @@ import "decred.org/dcrdex/client/intl"
+	"recipient": {T: "收件人" },
+	"block": {T: "区块" },
+	"timestamp": {T: "时间戳" },
+	"nonce": {T: "随机数" },
+	"Enable Privacy": {T: "启用隐私" },
+	"Privacy": {T: "隐私" },
+	"Enable": {T: "启用" },
+	"Disable Privacy": {T: "禁用隐私" },
+	"Configure Privacy": {T: "配置隐私" },
+	"Privacy active": {T: "隐私已启用" },
+	"Privacy off": {T: "隐私已关闭" },
+	"staking_disabled": {T: "票务购买已被<span id=\"extensionModeAppName\"></span>禁用。" },
+	"loading privacy status": {T: "加载隐私状态" },
+	"mixing_pw_prompt": {T: "输入密码以解锁钱包并启用隐私" },
+	"cspp_addr": {T: "CSPP服务器地址" },
+	"cspp_how": {T: "StakeShuffle创建的输出无法与先前的链上活动确定关联。虽然这种隐私方式下的输出金额仍然可见，但摧毁了可追溯性，意味着无法从公开数据推断出你的链上历史。" },
+	"privacy_intro": {T: "启用隐私时，您的所有资金将通过使用StakeShuffle协议的地址历史模糊服务发送。" },
+	"decred_privacy": {T: "Decred的隐私形式尤其强大，因为Decred钱包将隐私与质押结合，形成了一个始终较大的匿名集，这是隐私的关键特性。" },
+	"privacy_optional": {T: "隐私是完全可选的，可以随时禁用。启用隐私会产生额外的交易费用，但这些费用历来相对较低。" },
+	"privacy_unlocked": {T: "钱包在混合时必须保持解锁状态。" },
+	"bots_running_view_only": {T: "机器人正在运行。您处于仅查看模式。" },
+	"select_a_cex_prompt": {T: "选择一个交易所以启用套利" },
+	"Market not available": {T: "市场不可用" },
+	"bot_profit_title": {T: "选择您的利润阈值" },
+	"bot_profit_explainer": {T: "套利机器人只会尝试那些能够带来至少这个水平利润的交易。" },
+	"configure_cex_prompt": {T: "配置您的交易所API以启用套利功能。" },
+	"API Key": {T: "API 密钥" },
+	"API Secret": {T: "API 密钥" },
+	"Available": {T: "可用" },
+	"Locked": {T: "锁定" },
+	"profit_loss": {T: "利润/亏损" },
+		"base_change": {T: "基础变动" },
+	"quote_change": {T: "报价变动" },
+	"base_fees": {T: "基础费用" },
+	"quote_fees": {T: "报价费用" },
+	"run_logs": {T: "运行日志" },
+	"previous_run_logs": {T: "查看之前的运行日志" },
+	"previous_mm_runs": {T: "之前的市场制造运行" },
+	"start_time": {T: "开始时间" },
+	"end_time": {T: "结束时间" },
+	"logs": {T: "日志" },
+	"logs_for": {T: "日志：" },
+	"view_logs": {T: "查看日志" },
+	"swap_transaction": {T: "交换交易" },
+	"redeem_transaction": {T: "兑换交易" },
+	"refund_transaction": {T: "退款交易" },
+	"funding_transaction": {T: "资金交易" },
+	"transaction_id": {T: "交易ID" },
+	"credited_amt": {T: "贷方金额" },
+	"amt_received": {T: "收到金额" },
+	"dex_order_details": {T: "DEX订单详情" },
+	"cex_order_details": {T: "CEX订单详情" },
+	"deposit_details": {T: "存款详情" },
+	"withdrawal_details": {T: "取款详情" },
+	"base_filled": {T: "基础已填充" },
+	"quote_filled": {T: "报价已填充" },
+	"Details": {T: "详情" },
+	"Fee Allocation": {T: "费用分配" },
+	"Booking reserves": {T: "预定储备" },
+	"Swap funding": {T: "交换资金" },
+	"swaps": {T: "交换" },
+	"register_to_enable_mm": {T: `注册与<span data-tmpl="host"></span>启用交易` },
+	"No markets available": {T: "没有可用市场" },
+	"Arb": {T: "套利" },
+	"Choose Your Bot": {T: "选择你的机器人" },
+	"only_basic_mm_no_more_cexes": {T: "仅提供基础市场制造机器人。要启用套利选项，请配置您的交易所API。" },
+	"only_basic_mm_more_cexes": {T: "仅为此市场提供基础市场制造机器人。" },
+	"configure_more_cexes": {T: "配置其他交易所以最大化您的选择。" },
+	"Basic Market Maker": {T: "基础市场制造商" },
+	"mm_plus_arb": {T: "市场制造商 + 套利" },
+	"Basic Arbitrage": {T: "基础套利" },
+	"Fix errors": {T: "修复错误" },
+	"Commited": {T: "已承诺" },
+	"Required": {T: "必需" },
+	"OK": {T: "确定" },
+	"remote exchange gap": {T: "远程交易所差价", "Notes": "CEX订单簿上的差价" },
```

Comment:
Please remove "Notes" here, they are only for translators and only need be in the english version.
```suggestion
	"remote exchange gap": {T: "远程交易所差价"},
```

Reply:
ok